### ISSUE TITLE
Vulnerability: csv-parse - Regular Expression Denial of Service

### ISSUE THREAD

**Issue by buptliuhs** [jsforce/jsforce#925]
```
                       === npm audit security report ===

# Run  npm install csv-parse@4.4.6  to resolve 1 vulnerability
SEMVER WARNING: Recommended action is a potentially breaking change
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ csv-parse                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ csv-parse                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ csv-parse                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1171                            │
└───────────────┴──────────────────────────────────────────────────────────────┘


found 1 high severity vulnerability in 9460 scanned packages
  1 vulnerability requires semver-major dependency updates.
```

**Comment by apasrini** [comment]
Even i am getting this issue csv parse needs to be updated to version 4.4.6 or higher 
https://www.npmjs.com/advisories/1171

**Comment by jruizvaldiviaffdc** [comment]
I have the same issue, any idea when the module will be updated to use csv-parse 4.4.6 or higher?

**Comment by createthis** [comment]
@stomita what is the plan here?

**Comment by acrosman** [comment]
I am working on a simple PR to bump this to the current version. While NPM flag this as a breaking change since the current locked version is 1.3.3, I don't actually see any use in the JSForce code base that would actually break since the calls are pass through as best I can tell.  

However, the changes _could_ impact downstream users if they are using CSV-Parse in ways that have changed.

**Comment by apasrini** [comment]
@stomita Could you please check on this issue as it is turning high priority vulnerability issue for us. I see a PR from @acrosman Thanks for this.

**Comment by acrosman** [comment]
Fixed by #930

**Comment by keanedawg** [comment]
Should this be closed?

**Comment by buptliuhs** [comment]
Fixed in 1.9.3
